cms/static/js/factories/import.js

```diff
@@ -1,8 +1,14 @@
 define([
     'js/views/import', 'jquery', 'gettext', 'jquery.fileupload', 'jquery.cookie'
-], function(CourseImport, $, gettext) {
+], function(Import, $, gettext) {
     'use strict';
-    return function (feedbackUrl) {
+    return function (feedbackUrl, library) {
+        var dbError;
+        if (library) {
+            dbError = gettext('There was an error while importing the new library to our database.');
```

Comment:
Nit: Since the error messages only differ by "library" and "course", I'd parameterize them.